cid/helpers/quicksight/__init__.py

```diff
@@ -234,12 +243,88 @@ def _recoursive_add_view(view):
             logger.info(f'"{dashboard.name}" ({dashboardId}) discover complete')
             return dashboard
 
+    def ensure_exist_group(self, groupname='cid-owners', description='Created by Cloud Intelligence Dashboards'):
+        try:
+            group = self.identityClient.describe_group(
+                AwsAccountId=self.account_id,
+                Namespace='default',
+                GroupName=groupname
+            ).get('Group')
+        except self.client.exceptions.ResourceNotFoundException:
+            group = self.identityClient.create_group(
+                AwsAccountId=self.account_id,
+                GroupName=groupname,
+                Namespace='default',
+                description=description,
+            ).get('Group')
```

Comment:
We need to handle AccessDenied exception too

Reply:
fixed